### PR TITLE
neomutt: add module

### PR DESCRIPTION
**Pull request by rycee**
A development of #287 with support for the new email infrastructure.

### CODE REVIEW

File: modules/programs/neomutt.nix [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+          }
+      )
+    ];
+
+    home.packages = [ pkgs.neomutt ];
+
+    xdg.configFile."neomutt/neomuttrc".text =
+      let
+
+        gpgSection = ''
```
Comment (teto) [inline modules/programs/neomutt.nix]: maybe we should introduce a gpg submodule first :p

**Comment by teto** [comment]
(random comment about mutt config generation, for potential inspiration https://github.com/LukeSmithxyz/mutt-wizard )

**Comment by stites** [comment]
poke poke -- I would love to have this feature. has there been any movement on this?

**Comment by rycee** [comment]
@stites I pushed the latest version. I tried it out and it doesn't work very well, but if anybody wishes to continue it then they are most welcome. I won't continue further on this module.

**Comment by teto** [comment]
while I prefer alot as an MUA, I keep having (different) trouble with it so I suppose having a backup MUA would be nice. I'll try updating it.
I would like to have some feedback on what's best approach to support multiple accounts in neomutt. Seems like the approach for multiaccounts is to write a config per account then source all of them and provide mappings to switch between accounts/resource the files.

**Comment by rycee** [comment]
Obsolete since #1002 was merged.

**Pull request closed without merging**